<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>slfile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="myTreeList" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <column>
        <property name="text">
         <string>文件名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>目标位置</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>修改时间</string>
        </property>
       </column>
      </widget>
      <widget class="QWebView" name="webView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu_skin">
     <property name="title">
      <string>皮肤(&amp;V)</string>
     </property>
     <addaction name="defaultAction"/>
     <addaction name="bookAction"/>
     <addaction name="grayAction"/>
    </widget>
    <widget class="QMenu" name="menu_language">
     <property name="title">
      <string>语言</string>
     </property>
     <addaction name="cnAction"/>
     <addaction name="enAction"/>
     <addaction name="jpAction"/>
    </widget>
    <addaction name="fullScreenAction"/>
    <addaction name="showClassTreeAction"/>
    <addaction name="menu_skin"/>
    <addaction name="menu_language"/>
    <addaction name="separator"/>
    <addaction name="showSatebarAction"/>
    <addaction name="separator"/>
    <addaction name="customToolbarAction"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="importAction"/>
    <addaction name="exportAction"/>
    <addaction name="separator"/>
    <addaction name="saveAsAction"/>
    <addaction name="saveToMobiAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="userBookAction"/>
    <addaction name="homepageAction"/>
    <addaction name="separator"/>
    <addaction name="inviteAction"/>
    <addaction name="separator"/>
    <addaction name="checkNewAction"/>
    <addaction name="separator"/>
    <addaction name="bbsAction"/>
    <addaction name="aboutAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="optionAction"/>
    <addaction name="separator"/>
    <addaction name="viewLogAction"/>
    <addaction name="separator"/>
    <addaction name="pluginAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tool"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>选项 </string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>查看日志</string>
   </property>
  </action>
  <action name="importAction">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>文档另存为</string>
   </property>
  </action>
  <action name="saveToMobiAction">
   <property name="text">
    <string>另存到移动设备</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="fullScreenAction">
   <property name="text">
    <string>全屏</string>
   </property>
  </action>
  <action name="showClassTreeAction">
   <property name="text">
    <string>显示/隐藏分类树</string>
   </property>
  </action>
  <action name="defaultAction">
   <property name="text">
    <string>默认</string>
   </property>
  </action>
  <action name="bookAction">
   <property name="text">
    <string>书香</string>
   </property>
  </action>
  <action name="grayAction">
   <property name="text">
    <string>灰色</string>
   </property>
  </action>
  <action name="customToolbarAction">
   <property name="text">
    <string>自定义工具栏</string>
   </property>
  </action>
  <action name="showSatebarAction">
   <property name="text">
    <string>显示状态栏</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>插件管理</string>
   </property>
  </action>
  <action name="userBookAction">
   <property name="text">
    <string>用户手册</string>
   </property>
  </action>
  <action name="homepageAction">
   <property name="text">
    <string>个人首页</string>
   </property>
  </action>
  <action name="inviteAction">
   <property name="text">
    <string>邀请朋友</string>
   </property>
  </action>
  <action name="bbsAction">
   <property name="text">
    <string>用户论坛</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="checkNewAction">
   <property name="text">
    <string>检查新版本</string>
   </property>
  </action>
  <action name="optionAction">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="viewLogAction">
   <property name="text">
    <string>查看日志文件</string>
   </property>
  </action>
  <action name="pluginAction">
   <property name="text">
    <string>插件管理</string>
   </property>
  </action>
  <action name="cnAction">
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="enAction">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="jpAction">
   <property name="text">
    <string>日本語</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>myTreeList</class>
   <extends>QTreeView</extends>
   <header location="global">mytreelist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
